docs/source/en/model_doc/omnivore.mdx

```diff
@@ -0,0 +1,62 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Omnivore
+
+## Overview
+
+The Omnivore Model was proposed in [Omnivore: A Single Model for Many Visual Modalities](https://arxiv.org/abs/2201.08377) by Rohit Girdhar, Mannat Singh, Nikhila Ravi, Laurens van der Maaten, Armand Joulin, Ishan Misra.
+The main idea here is generalization of vision transformer model across different modalities. The authors propose a single model 
+for multiple classification tasks such as image classification, video classification and 3D scence classification.
+
+The abstract from the paper is the following:
+
+*Prior work has studied different visual modalities in isolation and developed separate architectures for recognition of images, 
+videos, and 3D data. Instead, in this paper, we propose a single model which excels at classifying images, videos, and single-view
+3D data using exactly the same model parameters. Our 'Omnivore' model leverages the flexibility of transformer-based architectures
+and is trained jointly on classification tasks from different modalities. Omnivore is simple to train, uses off-the-shelf standard
+datasets, and performs at-par or better than modality-specific models of the same size. A single Omnivore model obtains 86.0% on
+ImageNet, 84.1% on Kinetics, and 67.1% on SUN RGB-D. After finetuning, our models outperform prior work on a variety of vision
+tasks and generalize across modalities. Omnivore's shared visual representation naturally enables cross-modal recognition without
+access to correspondences between modalities. We hope our results motivate researchers to model visual modalities together.*
+
+Tips:
+
+- All released checkpoints were pre-trained and fine-tuned on three datasets [ImageNet-1k]((https://huggingface.co/datasets/imagenet-1k)), [Kinetics](https://www.deepmind.com/open-source/kinetics) and [SUN RGB-D](https://rgbd.cs.princeton.edu/) respectively. 
```

Comment:
```suggestion
- All released checkpoints were pre-trained and fine-tuned on three datasets [ImageNet-1k](https://huggingface.co/datasets/imagenet-1k), [Kinetics](https://www.deepmind.com/open-source/kinetics) and [SUN RGB-D](https://rgbd.cs.princeton.edu/) respectively. 
```

Reply:
Why is fine-tuning coming here? As far as I remember, they all were pretrained on these three datasets. The [official model zoo](https://github.com/facebookresearch/omnivore#model-zoo) (apart from Epic Kitchens) also does not enlist any fine-tuned checkpoints. 

Also, since we're providing the official sources for Kinetics and SUN I think it makes sense to also link the official website of ImageNet.

Reply:
Yes there's only 1 fine-tuned checkpoint released. It seems that this is just supervised learning on several datasets combined (multi-task learning).

Also, you can leave the link to ImageNet on the hub, the goal is to update the other links once they are also available on the hub.

Reply:
> It seems that this is just supervised learning on several datasets combined (multi-task learning).

The pre-training is supervised learning with multiple dataset using a shared backbone, yes! But interestingly, the training objective doesn't vary i.e., classification throughout. It's not like one head is doing classification, another one is doing regression, and so on. 

> Yes there's only 1 fine-tuned checkpoint released.

If we're not including the fine-tuned checkpoint then it makes sense to remove "fine-tuned" from here - `All released checkpoints were pre-trained and fine-tuned ...`.